runtime-modules/project-token/src/lib.rs

```diff
@@ -163,6 +164,15 @@ decl_storage! { generate_storage_info
 
         /// Platform fee (percentage) charged on top of each sale purchase (in JOY) and burned
         pub SalePlatformFee get(fn sale_platform_fee) config(): Permill;
+
+        /// Percentage threshold for deactivating the amm functionality
+        pub AmmDeactivationThreshold get(fn amm_deactivation_threshold) config(): Permill;
```

Comment:
This is the intended behavior.
The CC cannot close the AMM as he pleases. If the public buys 100 CRT through the AMM then the CC cannot close the AMM unless the public has sold back to the AMM at least 95 CRT, this is the intended behavior

Now, the question is: do we want to allow the CC to close the AMM at his discretion, maybe say allowing for a gracing period as suggested (to prevent CC to immediately lock out CRT holders from selling back their token)? @bedeho